fdbserver/TenantBalancer.actor.cpp

```diff
@@ -388,7 +388,8 @@ class MovementRecordMap {
 
 	ACTOR static Future<Void> saveRecordImpl(MovementRecordMap* self,
 	                                         Database db,
-	                                         Reference<const MutableRecord> mutableRecord) {
+	                                         Reference<const MutableRecord> mutableRecord,
+	                                         bool persistent = true) {
```

Comment:
I would recommend taking this argument as a timeout, that way we can express the case that we don't want to persist at all.

```suggestion
	                                         Optional<double> timeout = Optional<double>()) {
```